storage/migrations/mysql/5_scheduled_campaigns.sql

```diff
@@ -0,0 +1,15 @@
+-- +migrate Up
+
+CREATE TABLE IF NOT EXISTS `scheduled_campaigns`
```

Comment:
```suggestion
CREATE TABLE IF NOT EXISTS `campaign_schedules`
```

Reply:
@Dzalevski we need to change this on `shared/scheduled-campaigns` branch